pkgs/applications/networking/instant-messengers/hangups/default.nix

```diff
@@ -0,0 +1,88 @@
+{ pkgs, stdenv, python3 }:
+let 
+  packageOverrides = self: super:{
+    ConfigArgParse = python3.pkgs.buildPythonPackage rec {
+      src = python3.pkgs.fetchPypi {
+        pname = "ConfigArgParse";
+        version = "0.11.0";
```

Comment:
`version` has to be specified inside `super.ConfigArgParse.override` instead of inside the `src`. You can use `inherit version` here.